src/index.js

```diff
@@ -1,6 +1,15 @@
 const fibonacci = (n) => {
 
+  const array = [n];
```

Comment:
You rewrite this value. If you wanted an array with n elements, use `const array = new Array(n)`.